proof/refine/AARCH64/ArchRetype_R.thy

```diff
@@ -5130,6 +1641,7 @@ lemma corres_retype_region_createNewCaps:
                 init_arch_objects (APIType_map2 (Inr ty)) dev y n us x;
                 return x od)
             (createNewCaps ty y n us dev)"
+  supply APIType_map2_generic[simp del]
```

Comment:
This and `APIType_capBits_generic[simp del]` were needed a few times, I'm guessing to avoid warnings. Does that mean that the default simpset should be slightly tweaked inside `Arch` to avoid duplication?

Reply:
Happy to gamble on this one. If your gut feeling says this might be worthwile, let's try it. We can always go back and tweak things if we really care strongly about it.

Reply:
You have by far the most experience here so happy to go with your feeling as well. Would it make sense to mark any of them with a FIXME though, to make sure we look at it again once they're arch-split?